_includes/ICS-modal.html

```diff
@@ -0,0 +1,33 @@
+<script type="text/javascript" src="/assets/js/copy-url.js" />
```

Comment:
Could it happen that the script is included twice this way? That could lead to problems. I would rather include it in the main header and nowhere else.

Reply:
Quite unlikely but possible (e.g. if we add the ICS modal to a page with sharing options, or add sharing options to the events page). I included it here to avoid the js being loaded on every page, but I guess it's small enough not to be an issue.